### PR TITLE
docs: Ephemeral Container Doc

### PR DESCRIPTION
**Pull request by ashokdevtron**
Doc: Launching ephemeral containers

### CODE REVIEW

**Review by gireesh-devtron — APPROVED** [review]
LGTM

File: docs/user-guide/app-details/ephemeral-containers.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Using Ephemeral Containers on Devtron
+
+## Introduction
+
+Ephemeral container is a temporary disposable tool that helps you fix issues in your application containers. It provides a gateway (terminal) to perform process-level and OS-level debugging, without impacting the target container and its resources.
```
Comment (abhibhaw) [inline docs/user-guide/app-details/ephemeral-containers.md:5]: Definition is kinda incorrect / not clear. Please check the [official docs](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/), you can pick a few points from uses and what is section

File: docs/user-guide/app-details/ephemeral-containers.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+You can remove an ephemeral container using either **App Details** or **Resource Browser** (from the same screen you used to create the ephemeral container).
+
+{% hint style="info" %}
+You cannot use **App Details** or **Resource Browser** to remove an ephemeral container created using Kubernetes CLI
+{% endhint %}
+
+![Figure 1a: Removing Ephemeral Container from App Details](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/debugging-deployment-and-monitoring/delete-ec.jpg)
+
+![Figure 1b: Removing Ephemeral Container from Resource Browser](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/debugging-deployment-and-monitoring/delete-ec-2.jpg)
```
Comment (abhibhaw) [inline docs/user-guide/app-details/ephemeral-containers.md:76]: Kind of same type screenshots, you can remove one, better add screenshots for other step. Possible screenshots could be:

1. How to launch
2. Basic view 
3. Advance view
4. Delete container (Already Added)

Also try to reduce repeatibility in steps if possible, few points can be clubbed from Above different ways to access

**Review by abhibhaw — CHANGES_REQUESTED** [review]

**Review by abhibhaw — APPROVED** [review]
LGTM 🎉 🎉 Thanks @ashokdevtron

**Pull request merged**